docs/rsdl/rapid-rsdl-capability-mapping.mdx

```diff
@@ -0,0 +1,221 @@
+---
+id: rsdl-capability-mapping
+title: Capability semantics
+sidebar_label: RSDL API Capability semantics
+---
+
+> DRAFT
+> Initial Draft. Dec 2021
+
+Mapping RSDL capabilities to CSDL annotations
+
+## Delete
+
+```rsdl
+path /skus/{id} {
+    DELETE
+}
+```
+
+```xml
+ <Annotations Target="skus">
+  <Annotation Term="Org.OData.Capabilities.V1.DeleteRestrictions">
+    <Record Type="Org.OData.Capabilities.V1.DeleteRestrictionsType">
+      <PropertyValue Property="Deletable" Bool="true" />
+    </Record>
+  </Annotation>
+</Annotations>
+```
```

Comment:
Actually we need to produce the complement, i.e. `Deletable: false` for all service members that have no `path` allowing `DELETE`, same for all other "default" capabilities.